File: src/BukkitPE/level/generator/normal/biome/SandyBiome.php
<?php
namespace BukkitPE\level\generator\normal\biome;

use BukkitPE\block\Block;

abstract class SandyBiome extends NormalBiome{

	public function __construct(){
		$this->setGroundCover([
			Block::get(Block::SAND, 0),
			Block::get(Block::SAND, 0),
			Block::get(Block::SANDSTONE, 0),
			Block::get(Block::SANDSTONE, 0),
			Block::get(Block::SANDSTONE, 0),
		]);
	}
}

File: src/main/java/net/BukkitPE/level/generator/biome/SandyBiome.java
package net.BukkitPE.level.generator.biome;

import net.BukkitPE.block.Block;
import net.BukkitPE.block.BlockSand;
import net.BukkitPE.block.BlockSandstone;
import net.BukkitPE.level.generator.populator.PopulatorCactus;
import net.BukkitPE.level.generator.populator.PopulatorDeadBush;

/**
 * BukkitPE Project
 */
public abstract class SandyBiome extends NormalBiome {
    public SandyBiome() {

        PopulatorCactus cactus = new PopulatorCactus();
        cactus.setBaseAmount(2);

        PopulatorDeadBush deadbush = new PopulatorDeadBush();
        deadbush.setBaseAmount(2);

        this.addPopulator(cactus);
        this.addPopulator(deadbush);

        this.setGroundCover(new Block[]{
                new BlockSand(),
                new BlockSand(),
                new BlockSandstone(),
                new BlockSandstone(),
                new BlockSandstone()
        });
    }
}
